src/sentry/replays/tasks.py

```diff
@@ -104,3 +104,36 @@ def archive_replay(publisher: KafkaPublisher, project_id: int, replay_id: str) -
             }
         ),
     )
+
+
+@instrumented_task(
+    name="sentry.replays.tasks.publish_replay_viewed",
+    queue="replays.replay_viewed",
+    default_retry_delay=5,
+    max_retries=5,
+    silo_mode=SiloMode.REGION,
+)
+def publish_replay_viewed(project_id: int, replay_id: str, viewed_by_id: int) -> None:
```

Comment:
do we want to put some kind of rate limiting on this endpoint per user? what if a user views a replay 100000 times with a selenium script or something?

Reply:
Mm sounds smart. How do we implement this?